tutorials/dataframe/df025_RNode.C

```diff
@@ -1,38 +1,43 @@
 /// \file
 /// \ingroup tutorial_dataframe
 /// \notebook
-/// RNode is a generic type which represents any transformation node in the computation graph.
-/// This tutorial shows how to take advantage of the RNode class.
+/// Each RDataFrame object has its own type. It helps with performance,
+/// but sometimes it gets in the way of writing simple code that manages RDF objects.
+/// Luckily, every RDF object can be converted to the generic RNode type.
+/// This tutorial shows how to take advantage of RNode to easily manipulate RDataFrames.
 ///
 /// \macro_code
 /// \macro_output
 ///
-/// \date December 2018
+/// \date June 2020
 /// \author Danilo Piparo
+/// \author Enrico Guiraud
```

Comment:
If I remember correctly, doxygen wants `\authors ..., ...`. Not sure, though. Olivier could know this. You would have to check how this typesets once doxygen has been regenerated.

Reply:
if it's wrong, there are a lot of places that we need to fix :D